charts/lighthouse/values.yaml

```diff
@@ -92,7 +92,6 @@ webhooks:
 
   # webhooks.service -- Service settings for the webhooks controller
   service:
-    name: hook
```

Comment:
Still need to keep this for compat reasons. =)

Reply:
I’ll add an override mechanism and state in the docs that it’s there for compact reasons (a regular user should not need it).

Reply:
Both the name of the service and ingress matter ?

Reply:
Regrettably, yes - Jenkins X makes some hard assumptions on both.

Reply:
Ok, i adapted to chart templates but i think i'll need to change some config for the tests too.

Reply:
Actually, the ingress name can be changed - I forgot that jx has its own hook ingress. But yeah, the Tekton tests will need to change if the ingress name changes.

Reply:
We could look up the ingress by labels instead of relying on an hard coded name.